0051-options-local-pragma.rst

```diff
@@ -0,0 +1,106 @@
+Options Local Pragma
+====================
+
+.. proposal-number:: 
+.. ticket-url::
+.. implemented::
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/0>`_.
+.. sectnum::
+.. contents::
+
+We propose to add an opportunity to switch GHC-options locally. For instance, this code
+::
+ f = {-# OPTIONS_LOCAL -Wname-shadowing #-}
+   let a = () in
+   let a = 5 in
+     a + a
+get warning ``-Wname-shadowing`` to line
+::
+ let a = 5 in
+
+Motivation
+------------
+
+There are the following reasons to implement such pragma:
+
+1) **Local control of warnings**
```

Comment:
I have no clue how to use rst, but this formatting does not look right.

Reply:
I don't understand the problem. It looks like a bold text.

Reply:
This appears to be a list with a single element consisting of this one bold line. But what you probably want to do is make the entire following text a list item.

Reply:
I didn't find how to create a list where one point consists of some paragraphs, but fixed it another way.